src/site/xdoc/index.xml

```diff
@@ -15,346 +15,122 @@
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-            <p><img src="logo-call/james-logo-6-avatar.png"/></p>
-          </subsection>
-
-          <subsection name="Proposal 7">
-            <div id="james-logo-7-preview">
-              <p><img src="logo-call/james-logo-7.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-          <subsection name="Proposal 8">
-            <div id="james-logo-8-preview">
-              <p><img src="logo-call/james-logo-8.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-            <p><img src="logo-call/james-logo-8-avatar.png"/></p>
-          </subsection>
-
-          <subsection name="Proposal 9">
-            <div id="james-logo-9-preview">
-              <p><img src="logo-call/james-logo-9.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-          <subsection name="Proposal 10">
-            <div id="james-logo-10-preview">
-              <p><img src="logo-call/james-logo-10.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-        </div>
-
-<!-- 
-        <div id="tabs-5">
-  
-              <object width="480" height="385">
-                <param name="movie" value="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US"></param>
-                <param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param>
-                <embed src="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="480" height="385">
-                </embed>
-              </object>
-  
-        </div>
--->
+
+          <subsection name="Software Components">
+
+              <p>Developers looking for a modular mail platform on which to build can look
+                 at the modules and libraries used to compose James Server 3.0.</p>
+
+              <p><a href='http://james.apache.org/imap/'>Apache James IMAP</a> provides a flexible codec for IMAP,
+                 command processors and a sample data access layer. In combination with a socket layer,
+                 and a mailbox persistence, this library can be used to create an IMAP server.</p>
+
+              <p>The <a href='http://james.apache.org/mailets/'>Apache James Mailet project</a> collects products
```

Comment:
I think the idea is not to suppress things from Mailets project.
So what are your suggestions about details to add in the other components parts?